docs/topics/maven.md

```diff
@@ -1,21 +1,32 @@
 [//]: # (title: Maven)
 
-## Plugin and versions
+Maven is a build system that you can use to build and manage any Java-based project.
+
+## Configure plugin and versions
 
 The *kotlin-maven-plugin* compiles Kotlin sources and modules. Currently, only Maven v3 is supported.
 
-Define the version of Kotlin you want to use via a *kotlin.version* property:
+Define the version of Kotlin you want to use via the `kotlin.version` property:
 
 ```xml
 <properties>
     <kotlin.version>%kotlinVersion%</kotlin.version>
 </properties>
 ```
 
-## Dependencies
+### Use JDK 17
+
+To use JDK 17, in your `.mvn/jvm.config` file, add:
+
+```none
+--add-opens=java.base/java.lang=ALL-UNNAMED
+--add-opens=java.base/java.io=ALL-UNNAMED
+```
+
+## Set dependencies
 
 Kotlin has an extensive standard library that can be used in your applications.
-To use the standard library in your project, add the following dependency in the pom file:
+To use the standard library in your project, add the following dependency in your `pom.xml` file:
```

Comment:
NIT: Should we use "**to** your `pom.xml` file:" here?

Reply:
Nice suggestion. Fixed!